### DISCUSSION TITLE
Writing Match for EQY.

### DISCUSSION THREAD

**Discussion by Ajaymalolan** [YosysHQ/yosys#4469] [Q&A]
Hello, I was unable to find enough information about writing the match section of .eqy file while using EQY. I would like some detailed material or tutorial to learn about EQY from in detail. Please help me as soon as possible.

**Comment by KrystalDelusion** [comment]
Have you looked at the [relevant documentation section](https://yosyshq.readthedocs.io/projects/eqy/en/latest/config.html#match-sections)?

**Reply by Ajaymalolan** [reply]
Thanks for responding. Yes, I have referred it, it's not sufficient(and a bit confusing). I want to know what the exact syntax is, what it does and how can I write for my custom case.

**Reply by KrystalDelusion** [reply]
@jix are you able to offer any advice?

**Reply by jix** [reply]
I don't have anything beyond what @KrystalDelusion already linked, apart from perhaps the included examples at https://github.com/YosysHQ/eqy/tree/main/examples.

We have started work on a new version of EQY with the goal of making it much easier to use and making it require a lot less manual configuration, but this also means that the documentation of the current version won't be extended, as most of it will become obsolete with the new version.

**Reply by Ajaymalolan** [reply]
Thanks looking forward to the update.
